DistroLauncher/SetOnceNamedEvent.cpp

```diff
@@ -34,31 +35,31 @@ namespace Win32Utils
         if (event != nullptr) {
             CloseHandle(event);
             event = nullptr;
+            setOnce = false;
```

Comment:
No need to set variables in the destructor

Reply:
Indeed 🙈